Execute-SuperCollider-from-vim-config.md

```diff
@@ -0,0 +1,42 @@
+# Execute SuperCollider code from lua code in NeoVim config 
```

Comment:
As a more general observation, maybe this header (and title of the document) should be "Using scnvim from lua" or something along those line, since the examples are not really restricted to the users `init.vim` or `init.lua` configuration? 

If we want to keep the title (and focus) to configuration, maybe move the mappings example to the top of the file since it is the most relevant and also add the lua-heredoc delimiters to make it easier for users to copy-paste into their configs.

```
lua << EOF
-- lua code
EOF
```

Reply:
Thanks for the suggestions! 
should be fixed now